src/app/core/common-components/entity-select/entity-select.component.ts

```diff
@@ -156,7 +163,10 @@ export class EntitySelectComponent<E extends Entity> implements OnChanges {
   inputPlaceholder = this.loadingPlaceholder;
 
   allEntities: E[] = [];
+  relevantEntities: E[] = [];
```

Comment:
can you add a code comment or an even clearer variable name here? in what sense are they "relevant"? what's the difference to being "filtered"?